examples/WindowID.py

```diff
@@ -18,8 +18,8 @@ def main(self):
         caption,
         penguins,
         help,
-        Label.BackButton,
-        Label.NextButton
+        "Back", #Label.BackButton
```

Comment:
this is a bit step back, as `Label.BackButton` is translated and should be promoted.

Reply:
It was missing the import for the Label module (fixing)